src/SFML/Audio/SoundFileReaderMp3.cpp

```diff
@@ -99,8 +99,8 @@ bool SoundFileReaderMp3::check(InputStream& stream)
 ////////////////////////////////////////////////////////////
 SoundFileReaderMp3::SoundFileReaderMp3() : m_numSamples(0), m_position(0)
 {
-    std::memset(&m_io, 0, sizeof(m_io));
-    std::memset(&m_decoder, 0, sizeof(m_decoder));
+    m_io      = {};
+    m_decoder = {};
```

Comment:
Could we not use the constructor initializer list here?

Reply:
Good catch! I fixed that.

Reply:
@Bromeon does the fix match your expectations? 😄